Automattic/pocket-casts-ios
# Tracks: Fix a nan crash

This adds an isNan check to prevent a crash. 

## To test

1. Launch the app
2. Enable Tracks Loggin
3. Play any episode
4. Change the player position
5. ✅ Verify you see `🔵 Tracked: playback_seek ["seek_from_percent": PERCENT, "seek_to_percent": PERCENT, "source": "player"]`
6. In Xcode, open `PlaybackManager` and change line 391 to:
7. `analyticsPlaybackHelper.seek(from: 0, to: 0, duration: 0)`
8. Relaunch the app and change the player position
9. ✅ Verify the app doesn't crash and you don't see the log

## Checklist
